coralproject/talk
# Upgrade to Node 18 (Testing GHA CI pipelines)

## What does this PR do?

- Re-enables the reverted commit with the Node 18 changes

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

N/A

## How do I test this PR?

- Pull down branch
- Perform a production build using node `18.16.0` and npm `8.0.0`
- See that the build completes
    - Feel free to test Coral functionality too
- See that GHA build for this branch has also completed

## Were any tests migrated to React Testing Library?

No

## How do we deploy this PR?

- Merge into develop and target for another release
    - May want to release this on its own as its own release as it is upgrading all of node from `v14` to `v18`
